pyproject.toml

```diff
@@ -1,143 +1,186 @@
+[build-system]
+build-backend = "poetry_dynamic_versioning.backend"
+requires = [ "poetry-core>=1", "poetry-dynamic-versioning>=1,<2" ]
+
 [tool.poetry]
 name = "coSMicQC"
-version = "0.0.10.post7.dev0+d8d16e9"
+version = "0.0.0"
```

Comment:
I feel like this loves to constantly change lol I can't remember if this is correct so I will highlight this section for you to double check.